integration/confidential/lib.sh

```diff
@@ -55,6 +55,13 @@ switch_image_service_offload() {
 #			      `kata-runtime env` and export its value.
 #
 switch_measured_rootfs_verity_scheme() {
+	# In the case of IBM Secure Execution for Linux, the ultravisor (trusted IBM zSystems CPU firmware),
```

Comment:
I'm a bit torn on the check being in `switch_measured_rootfs_verity_scheme`. On the onehand having it in just a single place is good, but I don't know if it would be obvious to anyone adding tests to `agent_image.bats`?

I think it is fine as is, just thinking out loud a bit!

Reply:
I agree with your thought. (I was thinking it actually :wink:) It would be something to be relocated based on others' feedback. Let's see what others think about this placement.